packages/jest-config/src/normalize.js

```diff
@@ -132,12 +132,21 @@ const setupBabelJest = (options: InitialOptions) => {
 
     if (customJSPattern) {
       const customJSTransformer = transform[customJSPattern];
+      if (Array.isArray(customJSTransformer)) {
```

Comment:
this part has changed a bit on master, mind rebasing and updating here? (some typescript stuff added)